Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -173,6 +175,7 @@ def scan_repositories(repos, person, args)
         commits:       GitHub.repo_commit_count_for_user(repo_full_name, person, args),
         coauthorships: git_log_trailers_cmd(T.must(repo_path), person, "Co-authored-by", args),
         signoffs:      git_log_trailers_cmd(T.must(repo_path), person, "Signed-off-by", args),
+        reviews:       GitHub.count_issues("", is: "pr", repo: repo_full_name, reviewed_by: person),
```

Comment:
Definitely wouldn't have been blocking: is there any way to query the specific type of reviews?

Reply:
A query like `is:pr reviewed-by:#{user} review:approved` works to get approvals by the user, but we can't combine `review:approved` and `review:changes_requested`. So I thought that just `reviewed-by:#{user}` was safest to encompass everything.

Reply:
@issyl0 I'd consider filtering this to just `review:approved` as it'll get fewer results, requires write access (i.e. "maintainer duties") and is required to move things forward. Thoughts?